### PR TITLE
Adding files to create a Debian distribution package

### PR DESCRIPTION
**Pull request by skarbat**

### CODE REVIEW

**Comment by alex5imon** [comment]
@pazdera can you have a look as well?

File: debian/control [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+Maintainer: Team Kano <dev@kano.me>
+Section: x11
+Priority: optional
+Standards-Version: 1.1.0
+Build-Depends: libx11-dev, xorg-dev, xorg-dev, xutils-dev, x11proto-dri2-dev, libltdl-dev, libdrm-dev, debhelper (>=9.0.0)
+
+Package: xf86-video-fbturbo-improved
+Architecture: any
+Depends: libkdesk-dev (= ${binary:Version}), ${misc:Depends}, ${shlibs:Depends}, libstartup-notification0, libxss1
+Provides: xf86-video-fbturbo-improved
```
Comment (pazdera) [inline debian/control]: A very small thing: Is this line necessary?

File: debian/changelog [hunk 2]
```diff
@@ -0,0 +1,5 @@
+xf86-video-fbturbo-improved (1.1-00) unstable; urgency=low
```
Comment (pazdera) [inline debian/changelog]: This is more of my OCD, could you change the version just to **1.1-1**? It doesn't really matter, but that's the convention we have with the other packages.

**Comment by skarbat** [comment]
thanks for your detailed followup @pazdera Radek!

**Comment by pazdera** [comment]
Thanks for the fixes @skarbat! It looks solid to me.

Please merge :shipit:, I'm going to test it and add to peldins/updater scenario.

**Comment by skarbat** [comment]
Thanks @radek, worth doing a re-run of ```gtkperf```` because I enabled the ```hwcursor``` module flag.

**Pull request merged**